src/scripts/find_dadis_transboundary_ids.py

```diff
@@ -0,0 +1,221 @@
+import argparse
+import csv
+from typing import TextIO
+
+import pandas as pd
+
+from dadis_client import DadisClient
+
+
+def full_matching_workflow(
+    input_filename: str, output_filename: str, dadis_api_key: str
+) -> pd.DataFrame:
+    """
+    Perform the full matching workflow:
+
+    - Read VBO data from input_filename
+    - Match to DADIS to get dadis_transboundary_id
+    - Save to a new TSV file at output_filename
+    """
+    client = DadisClient(api_key=dadis_api_key)
+    vbo_data = read_vbo_data(input_filename)
+    matched_breeds = match_vbo_breeds(vbo_data=vbo_data, client=client)
+
+    print(f"Writing output file to {output_filename}:")
+    output_file = create_output_tsv(
+        input_filename=input_filename,
+        output_filename=output_filename,
+        extra_cols=["dadis_transboundary_id"],
+    )
+    matched_breeds.to_csv(output_file, sep="\t", index=False, header=False)
+    output_file.close()
+    print("Output written.")
+    return matched_breeds
+
+
+def read_vbo_data(filename: str) -> pd.DataFrame:
+    df = pd.read_table(filename, skiprows=[1]).convert_dtypes()
+    print("Dropping duplicate entry for: VBO:0000991 - Icelandic Horse/Iceland Pony")
```

Comment:
This should not be in the code.. Let @sabrinatoro remove this record manually

Reply:
Yes, just doing this to get the code working initially, but I don't want to keep one-off fixes like this in the code.